notificationsManager/Operators/Commands/UpdateOperators.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace NotificationsManager\Operators\Commands;
+
+use Exception;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Http as HttpClient;
+use NotificationsManager\Operators\OperatorsService;
+
+class UpdateOperators extends Command
+{
+    protected $signature = 'update:operators';
+    protected $description = 'Actualiza los datos de los operarios';
+    private HttpClient $httpClient;
+
+    private OperatorsService $operatorsService;
+
+    public function __construct(HttpClient $httpClient, OperatorsService $operatorsService)
+    {
+        parent::__construct();
+        $this->httpClient = $httpClient;
+        $this->operatorsService = $operatorsService;
+    }
+
+    public function handle(): int
+    {
+        $url = 'https://api.extexnal.com/operators/?sequence_number=12341234';
+        $operatorData = [
+            [
+                'sequenceNumber' => 1510105,
+                'journalEntryType' => 'UP',
+                'customerId' => 26,
+                'id' => 2,
+                'name' => '654654',
+                'surname1' => '',
+                'surname2' => '',
+                'phone' => 0,
+                'email' => '',
+                'orderNotifications' => false,
+                'orderNotificationEmail' => '',
+                'orderNotificationByEmail' => false,
+                'orderNotificationBySms' => false,
+                'orderNotificationByPush' => false,
+                'deleted' => true,
+                'object' => 'SALQ9U',
+                'objectSchema' => 'IQSFCOMUN'
+            ]
+        ];
+
+        $this->httpClient::fake([
+            $url => $this->httpClient::response($operatorData, 200)
+        ]);
+
+        try {
+            $response = $this->httpClient::get($url);
+
+            if ($response->failed()) {
+                throw new Exception($response, 500);
+            }
+
+            $operatorsInfo = $response->json();
+
+            $this->operatorsService->updateOperator($operatorsInfo[0]);
```

Comment:
Lo que te decía en un comentario anterior
```suggestion
            $this->updateOperatorsService->update($operators);
```